pkg/bundle/client.go

```diff
@@ -65,7 +65,7 @@ func (bc *bundleClient) GetActiveBundle(ctx context.Context, clusterName string)
 		return nil, err
 	}
 
-	if pbc.Spec.ActiveBundle == "" {
+	if len(pbc.Spec.ActiveBundle) < 1 {
```

Comment:
```suggestion
	if len(pbc.Spec.ActiveBundle) == 0 {
```

This seems to be a more consistent write-up with the `pkg.bundle/manager.go` file, and easier to understand IMO since you don't really expect the length of bundle to ever go below 0 right?

Reply:
I agree with what Eric says here. I was thinking there must be some weird behavior on the comparator that triggered this change. If there are no such constraint, I would vote for `""` as well.